make/Images.gmk

```diff
@@ -98,16 +111,48 @@ $(eval $(call SetupExecute, jlink_jdk, \
     WARN := Creating jdk image, \
     DEPS := $(JDK_JMODS) $(BASE_RELEASE_FILE) \
         $(call DependOnVariable, JDK_MODULES_LIST, $(JDK_IMAGE_SUPPORT_DIR)/_jlink_jdk.vardeps), \
-    OUTPUT_DIR := $(JDK_IMAGE_DIR), \
+    OUTPUT_DIR := $(JDK_LINK_OUTPUT_DIR), \
     SUPPORT_DIR := $(JDK_IMAGE_SUPPORT_DIR), \
-    PRE_COMMAND := $(RM) -r $(JDK_IMAGE_DIR), \
+    PRE_COMMAND := $(RM) -r $(JDK_LINK_OUTPUT_DIR), \
     COMMAND := $(JLINK_TOOL) --add-modules $(JDK_MODULES_LIST) \
-        $(JLINK_JDK_EXTRA_OPTS) --output $(JDK_IMAGE_DIR) \
+        $(JLINK_JDK_EXTRA_OPTS) --output $(JDK_LINK_OUTPUT_DIR) \
         $(JLINK_DISABLE_WARNINGS), \
 ))
 
 JLINK_JDK_TARGETS := $(jlink_jdk)
 
+ifeq ($(JLINK_PRODUCE_RUNTIME_LINK_JDK), true)
+
+  JDK_RUN_TIME_IMAGE_SUPPORT_DIR := $(SUPPORT_OUTPUTDIR)/images/jlink-run-time
+  JDK_JMODS_DIFF := $(JDK_RUN_TIME_IMAGE_SUPPORT_DIR)/jimage_packaged_modules_diff.data
+  JLINK_RUNTIME_CREATE_EXTRA += --create-linkable-runtime=$(JDK_JMODS_DIFF)
```

Comment:
My understanding is that the `--create-linkable-runtime` is the primary argument for the jlink invocation in jlink_runtime_jdk. As such, I think it would be much better if this was inlined in place in the jlink_runtime_jdk COMMAND. The "extra" makes it sound like it is optional. 

In contrast, the value of JLINK_RUNTIME_CREATE_EXTRA that is set above for JLINK_KEEP_PACKAGED_MODULES is indeed "extra".

Reply:
Your understanding is correct. Will fix.